contrib/flo-bigquery/src/main/java/com/spotify/flo/contrib/bigquery/DefaultBigQueryClient.java

```diff
@@ -75,25 +71,6 @@ public TableId createStagingTableId(TableId tableId, String location) {
     return randomStagingTableId(tableId, location);
   }
 
-  @Override
-  public BigQueryResult query(QueryRequest queryRequest) {
```

Comment:
Do we want to bring back this method?

Reply:
Not unless it's needed?

Reply:
Not really. Even query is wrapped in a `JobInfo`, and `QueryRequest` should not be used anymore.